### DISCUSSION TITLE
How to get an equirectangular image?

### DISCUSSION THREAD

**Discussion by lambdald** [OpenStitching/stitching#201] [Q&A]

**Comment by lukasalexanderweber** [comment]
The default warper is equirectangular as far as I know (`spherical` warper projected on a plane)

**Reply by lambdald** [reply]
The stitched panorama does not meet the equirectangular projection that the width of the image is equal to twice the height.

**Reply by lukasalexanderweber** [reply]
You can warp the image content to match the 2:1 ratio

**Reply by lukasalexanderweber** [reply]
Have you got your desired result? If yes you can share your sollution with us :)

**Reply by lambdald** [reply]
Thanks for your solution! The stitched images are usually wider, so I filled in the missing height in black to match the 2:1 ratio, which can barely be used as a equirectangular image. However, The stitched image has imaging errors and is not a true panoramic image.
